app/models/user.rb

```diff
@@ -141,9 +138,9 @@ class User < ApplicationRecord
   validates :subscribed_to_user_subscriptions_count, presence: true
   validates :unspent_credits_count, presence: true
   validates :username, length: { in: 2..USERNAME_MAX_LENGTH }, format: USERNAME_REGEXP
-  validates :username, presence: true, exclusion: { in: ReservedWords.all, message: MESSAGES[:invalid_username] }
+  validates :username, presence: true, exclusion: { in: ReservedWords.all, message: :invalid_username }
   validates :username, uniqueness: { case_sensitive: false, message: lambda do |_obj, data|
-    "#{data[:value]} is taken."
+    I18n.t("models.user.is_taken", data_value: (data[:value]))
```

Comment:
`data_value` is a pretty opaque key because it just mirrors the actual variable name. Shouldn't we call it `username` since that's what it actually contains?